migrations/versions/0499cef6b57a_remove_api_keys.py

```diff
@@ -0,0 +1,38 @@
+"""remove api keys
+
+Revision ID: 0499cef6b57a
+Revises: 70bcddc9fffa
+Create Date: 2022-09-25 14:25:18.790980
+
+"""
+import sqlalchemy as sa
+import sqlmodel  # noqa: F401
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0499cef6b57a"
+down_revision = "70bcddc9fffa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("apikey")
+    op.alter_column("reciperun", "status", existing_type=sa.VARCHAR(), nullable=False)
```

Comment:
Why do we alter `reciperun.status` here?

Reply:
AFAICT we (I?) must've set a default on `reciperun.status` and then forgot to run a migration. So if IIUC this is picking up a change to this field since the last migration was run, but this change is not part of this PR. I am inclined to just leave this in here, since it's a small change, and one I think we want (making `reciperun.status` non-nullable).